.github/workflows/arm64.yml

```diff
@@ -0,0 +1,308 @@
+name: arm64
+
+on:
+  pull_request:
+  push:
+    paths-ignore:
+      - '.github/ISSUE_TEMPLATE/**'
+    branches:
+      - master
+
+env:
+  RUSTFLAGS: -Dwarnings
+
+jobs:
+  check:
+    runs-on: windows-2019
+
+    strategy:
+      matrix:
+        include:
+          - version: stable
+            target: aarch64-pc-windows-msvc
+          - version: nightly
+            target: aarch64-pc-windows-msvc
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Update toolchain
+        run: rustup update --no-self-update ${{ matrix.version }} && rustup default ${{ matrix.version }}-x86_64-pc-windows-msvc
+      - name: Add toolchain target
+        run: rustup target add ${{ matrix.target }}
+      - name: Fix environment
+        uses: ./.github/actions/fix-environment
+      - name: Clean
+        run:  cargo clean
+      - name: Test riddle
+        run:  cargo test -p riddle --no-run --target ${{ matrix.target }}
```

Comment:
Could you have re-used test.yml and made `--no-run` part of the matrix?

Reply:
That was the goal but that would require `--target` and then it runs afoul of https://github.com/rust-lang/cargo/issues/9661 and none of the DLL "cdylib" tests would succeed.